### ISSUE TITLE
Extractor error: Sequence contains no elements

### ISSUE THREAD

**Issue by kaytrip** [Azure/azure-api-management-devops-resource-kit#468]
When running the Extractor in the CLI I get the following error:

```
Extracting products from my-api API:
Error occured: Sequence contains no elements
```

I've seen it fail on Extracting tags as well, when exporting other API's.

**Last part of the log**
```
 - Operation policy found for xxxxxx operation
'xxxxxx' Operation found
 - Operation policy found for xxxxxx operation
'xxxxxx' Operation found
API policy found
------------------------------------------
Extracting API version sets from service
------------------------------------------
Extracting authorization servers from service
------------------------------------------
Extracting loggers from service
------------------------------------------
Extracting products from service
'my-other-api-product' Product found
'starter' Product found
 - Product policy found for starter product
'unlimited' Product found
1 APIs found ...
------------------------------------------
Extracting products from my-api API:
Error occured: Sequence contains no elements
```

**APIM configuration**
In my APIM I have a few API's with policies and one defined a product, but this product is not used for all API's. Some API's are accessible without a product.

**First hunch**
Perhaps something in the hierarchy when a product is used but not for all API's? But I also get it at tags when exporting two API's:

```
------------------------------------------
Extracting products from my-api API:
2 APIs found ...
------------------------------------------
Extracting tags from my-other-api API:
'xxxxxx' Operation found
Error occured: Sequence contains no elements
Sequence contains no elements
```

**Comment by fortunkam** [comment]
I think this is a duplicate of #466 (PR #467 is pending to resolve it)

**Comment by travich** [comment]
I have the same issue - do you know the latest stable branch I could go back to?

**Comment by fortunkam** [comment]
Not an ideal solution but until the PR is actioned you could try my patched fork (https://github.com/fortunkam/azure-api-management-devops-resource-kit)

**Comment by travich** [comment]
@fortunkam I've tried your branch and get the same error.

**Comment by miaojiang** [comment]
Closed via #467
